data/schema/core/addons.cfg

```diff
@@ -126,6 +126,7 @@
 	{SIMPLE_KEY filter_formula string}
 	{DEFAULT_KEY first_time_only bool yes}
 	{DEFAULT_KEY delayed_variable_substitution bool no}
+    {DEFAULT_KEY priority s_real 0.0}
```

Comment:
For consistency with the rest of the file, this line should use a tab instead of spaces. Or alternatively, we fix #5709, but that causes merge conflicts with several open PRs (probably, I haven't checked how may of the current PRs touch the schema).